flint.ui/src/views/Landing.vue

```diff
@@ -64,101 +34,149 @@
-          <br>
-          <br>
-          <router-link to="/gcbm/dashboard">
-            <button
-              class="
-                bg-white
-                text-gray-800
-                hover:bg-green-200
-                font-semibold
-                py-2
-                px-4
-                border border-gray-400
-                hover:border-green-600
-                rounded
-                shadow
-              "
-            >
-              GCBM
-            </button>
-          </router-link>
+            <router-link to="/flint/configurations/point">
+              <button
+                class="
+                  bg-white
+                  text-lg text-gray-800
+                  hover:bg-green-200
+                  font-semibold
+                  py-10
+                  px-20
+                  border border-gray-400
+                  hover:border-green-600
+                  rounded
+                  shadow
+                "
+              >
+                <div data-v-step="1">POINT</div>
+              </button>
+            </router-link>
+
+            <router-link to="/flint/configurations/rothc">
+              <button
+                class="
+                  bg-white
+                  text-lg text-gray-800
+                  hover:bg-green-200
+                  font-semibold
+                  py-10
+                  px-20
+                  border border-gray-400
+                  hover:border-green-600
+                  rounded
+                  shadow
+                "
+              >
+                <div data-v-step="2">ROTHC</div>
+              </button>
+            </router-link>
+
+            <router-link to="/gcbm/dashboard">
+              <button
+                class="
+                  bg-white
+                  text-lg text-gray-800
+                  hover:bg-green-200
+                  font-semibold
+                  py-10
+                  px-20
+                  border border-gray-400
+                  hover:border-green-600
+                  rounded
+                  shadow
+                "
+              >
+                <div data-v-step="3">GCBM</div>
+              </button>
+            </router-link>
+          </div>
+          <v-tour name="MyTour" :steps="steps" :options="myOptions"></v-tour>
         </div>
       </div>
     </div>
   </div>
 </template>
+
 <script>
 export default {
   components: {}
 }
 </script>
+
+<script>
+export default {
+  name: 'MyTour',
+  data() {
+    return {
+      myOptions: {
+        useKeyboardNavigation: true,
+        labels: {
+          buttonSkip: 'Skip tour',
+          buttonPrevious: 'Previous',
+          buttonNext: 'Next',
+          buttonStop: 'Finish'
+        }
+      },
+      steps: [
+        {
+          target: '[data-v-step="1"]',
+          content:
+            'Point is a sample run of FLINT. To learn how to run the simulation using the FLINT UI dashboard follow the tour.',
```

Comment:
I would move the second this sentence to a new prompt (later), and then change the text to 

> Point is a simple three-pool example of a stock and flow modell running in FLINT